docs/user-guide/layer-settings.md

```diff
@@ -83,6 +83,9 @@ In particular, the user is allowed to:
 
 !!!Warning
     The *Format* and *Layer tile size* options are available only for the layers added from CSW and WMS catalog sources.
+    
+!!!note
+    Cached tiles works with WMS only with standard gridsets (exponential). When MapStore uses custom scales, the tile cache can not be used.
```

Comment:
```suggestion
    Cached tiles works with WMS only with standard gridsets (exponential). When MapStore uses custom scales, the tile cache can not be used. This because WMS TileGrid for OpenLayers (internal MapStore mapping library) and the TileGrid generated by GeoServer have different origins (top-left vs bottom-left). While for exponential resolutions the top left remains fixed (because there is only an integer number of tiles in the interval between top-left and bottom-left corner of the bbox, 1, 2, 4, 8 ....) with custom resolutions the number of tiles is not integer, so the coordinate of the tile of top-left may vary across scales. More details about this limitation in this [issue](https://github.com/geosolutions-it/MapStore2/issues/9025) on GitHub.
```